doc/source/developer/debug.md

```diff
@@ -41,17 +41,17 @@ The procedure from download to build is the following:
 ```console
 $ mkdir -p ~/work/
 $ cd ~/work/
-$ wget https://downloads.mariadb.org/rest-api/mariadb/11.4.3/mariadb-11.4.3.tar.gz
-$ tar xf mariadb-11.4.3.tar.gz
-$ rm -rf mariadb-11.4.3/storage/mroonga
+$ wget https://downloads.mariadb.org/rest-api/mariadb/11.4.4/mariadb-11.4.4.tar.gz
```

Comment:
We can use https://downloads.mariadb.org/rest-api/mariadb/ to detect the latest MariaDB.

Reply:
Thanks. I will think how to handle the response from https://downloads.mariadb.org/rest-api/mariadb/.